service/kernel/data_management/data_conversion_cpu.cpp

```diff
@@ -25,6 +25,99 @@ namespace data_management
 {
 namespace internal
 {
+/* only for AVX512 architecture with using intrinsics */
+#if defined(__INTEL_COMPILER)
+template <typename T>
+void vectorCopyInternal()
+{
+    vectorCopyInternal<T>();
+}
+
+template <>
+void vectorCopyInternal<float>(const size_t nrows, const size_t ncols, void * dst, void const * ptrMin, DAAL_INT64 * arrOffsets)
+{
+    float * pd           = (float *)dst;
+    float const * pmin   = (float *)ptrMin;
+    char const * ptrByte = (char *)ptrMin;
+
+    const size_t nColSize = ncols - ncols % 8;
+
+    for (size_t i = 0; i < nrows; ++i)
+    {
+        for (size_t j = 0; j < nColSize; j += 8)
+        {
+            __m256 ps = _mm512_i64gather_ps(*((__m512i *)&arrOffsets[j]), pmin + i, 1);
+            _mm256_storeu_ps(pd + i * ncols + j, ps);
+        }
+        for (size_t j = nColSize; j < ncols; ++j)
+        {
+            char const * pc   = ptrByte + arrOffsets[j];
+            float * pi        = (float *)(pc) + i;
+            pd[i * ncols + j] = *pi;
+        }
+    }
+}
+
+template <>
+void vectorCopyInternal<double>(const size_t nrows, const size_t ncols, void * dst, void const * ptrMin, DAAL_INT64 * arrOffsets)
+{
+    double * pd          = (double *)dst;
+    double const * pmin  = (double *)ptrMin;
+    char const * ptrByte = (char *)ptrMin;
+
+    const size_t nColSize = ncols - ncols % 8;
+
+    for (size_t i = 0; i < nrows; ++i)
+    {
+        for (size_t j = 0; j < nColSize; j += 8)
+        {
+            __m512d p = _mm512_i64gather_pd(*((__m512i *)&arrOffsets[j]), pmin + i, 1);
+            _mm512_storeu_pd(pd + i * ncols + j, p);
```

Comment:
This instruction was not defined with usage icc180. Problem happened on CEV build of 2020 mnt with icc180 compiler. Please check it.

Reply:
It's release branch

Reply:
the same code was merged to rls/daal-2020-mnt if I`m not mistaken.

Reply:
yes

Reply:
according to logs build for icc180 is successful